store/src/pmmr.rs

```diff
@@ -140,6 +140,10 @@ impl<T: PMMRable> Backend<T> for PMMRBackend<T> {
 		}
 	}
 
+	fn n_leafs(&self) -> u64 {
+		self.leaf_set.len() as u64
```

Comment:
This works for a "prunable" MMR like the output and rangeproof MMR where we remove (and then eventually prune) leaf positions. But I don't think this will work correctly for other MMRs like the kernel MMR and the header MMR where we only ever append and never remove. I don't think we use the `leaf_set` in these MMRs as we do not need to write this additional file to disk. _All_ leaves are "not removed" in the case of kernels, for example.